src-docs/src/views/form_controls/display_toggles.js

```diff
@@ -199,7 +199,6 @@ DisplayToggles.propTypes = {
   canAppend: PropTypes.bool,
   canInvalid: PropTypes.bool,
   extras: PropTypes.arrayOf(PropTypes.node),
-  // Manually building the spacer array to avoid having to import Spacer into codesandbox
```

Comment:
So if this was done manually because of codesandbox and we're no longer supporting codesandbox, couldn't this be generated automatically?

Reply:
Yes...? 😆 

I think my thought process was along the lines of, what if the comment was only the _original_ reason it was done this way, but over time other things relied on that behavior without updating the comment.

But I'll try updating it to be more DRY.